spec/services/accredited_provider/importer_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+RSpec.describe AccreditedProvider::Importer do
+  subject { described_class.call }
+
+  let(:existing_provider) { create(:provider) }
+
+  before do
+    next_year = Time.current.next_year.year
+
+    stub_request(
```

Comment:
One for the team, I suppose, what do we think about using something like [VCR](https://github.com/vcr/vcr) so we don't have to maintain these stubs within tests?